robottelo/ui/locators/common.py

```diff
@@ -26,11 +26,9 @@
     "alert.error": (
         By.XPATH, "//div[contains(@class, 'alert-danger')]"),
     "alert.success_sub_form": (
-        By.XPATH,
-        "//div[@ng-hide]//div[contains(@class, 'alert-success')]"),
+        By.XPATH, "//div[contains(@bst-alert, 'success')]"),
```

Comment:
Looks like after nutapane was removed, there is no more such entity as subform alert and we can use only `alert.success` instead.